canonical-websites/partners.ubuntu.com
# Move everything to cms

The partnerwebsite app was pointless. I'm moving its settings etc. to the `cms` app. This shouldn't affect any functionality.

Fixes #105 

QA
---

``` bash
./run
```

Check the site and the admin area behave exactly as it used to.
